prlworkflows/sqs.py

```diff
@@ -301,18 +296,20 @@ def enumerate_sqs(structure, subl_model, endmembers=True, scale_volume=True, ski
     # error checking
     if len(subl_model) != len(structure.sublattice_model):
         raise ValueError('Passed sublattice model ({}) does not agree with the passed structure ({})'.format(subl_model, structure.sublattice_model))
-    if np.any([len(subl) for subl in subl_model] < [len(subl) for subl in structure.sublattice_model]):
-        if skip_on_failure:
-            return []
-        raise ValueError('The passed sublattice model ({}) is of lower order than the passed structure supports ({})'.format(subl_model, structure.sublattice_model))
     possible_subls = []
     for subl, abstract_subl in zip(subl_model, structure.sublattice_model):
-        if endmembers:
-            subls = itertools.product(subl, repeat=len(abstract_subl))
-        else:
-            subls = itertools.permutations(subl, r=len(abstract_subl))
+        subls = itertools.product(subl, repeat=len(abstract_subl))
         possible_subls.append(subls)
     # TODO: possibly want combinations rather than permutations because [['Cu', 'Mg']] might == [['Mg', 'Cu']]
     unique_subl_models = itertools.product(*possible_subls)
+
     # create a list of concrete structures with the generated sublattice models
-    return [structure.get_concrete_sqs(model, scale_volume) for model in unique_subl_models]
+    result = []
```

Comment:
Kind of unclear to the reader what `result` and `non_degenerate` really mean. 

Maybe better names would be `unique_sqs` instead of `result` and `unique_configurations_occupancies` or something to that effect.